charts/asset-manager/values.yaml

```diff
@@ -71,6 +71,9 @@ clamResources:
     cpu: 500m
     memory: 1Gi
 
+# get NFS server from AWS console, overridden in ArgoCD config.
```

Comment:
Helm value comments follow the same convention as Go comments in that they start with the name of the variable (mostly for the benefit of automatic doc-generating tools).
```suggestion
# nfsServer is the address of the NFSv4 (or Amazon EFS) server where uploaded
# assets files are stored temporarily while awaiting virus scanning. Required.
# Overridden in ArgoCD config.
```

Reply:
👍